### ISSUE TITLE
Gallery disappering while inside

### ISSUE THREAD

**Issue by wollo77** [paulkim26/collab-museum#191]
Standing in the gallery, working on a puzzle, the whole gallery suddenly disappeared.
No idea what happened there. Could this have been caused by reaching the end of some music logic?
![image](https://github.com/paulkim26/collab-museum/assets/135688731/29552650-6381-465c-be30-e4227de001c9)

**Comment by FranchNotFrench** [comment]
Hmm I hope I didn't break something. I didn't intentionally connect any music to visibility logic and I don't recall doing anything with visibility in general last night other than adding the new karaoke Display to the Debug viz activator.

**Comment by wollo77** [comment]
> Hmm I hope I didn't break something. I didn't intentionally connect any music to visibility logic and I don't recall doing anything with visibility in general last night other than adding the new karaoke Display to the Debug viz activator.

Yeah, the music was just a wild guess because it happened completely out of the blue, so I suspected some long-running timer...

**Comment by FranchNotFrench** [comment]
There is a lock in each tunnel connected to the music (should be the same locks Chazzy had connected to the test Atrium loop item which no longer exists) which are called Viz Render locks but I wouldn't expect the music to alter lock pin values unless it is a bug. Feel free to delete that music linkage for now though.

**Comment by FranchNotFrench** [comment]
With PR #192, I deleted the connection between the section locks that are
labeled something like "viz render" from the new music logic items by the
elevator. Which may not solve the Gallery viz issue but should hopefully
rule out music (and me!) as the culprit. :)

In general I have no idea how section viz logic is working now. My initial
version was pretty straightforward but also didn't account for all the
interior/exterior logic we decided to use. I look at all the logic objects
in the tunnels and just think to myself, "I hope somebody understands this"
:D

**Comment by FranchNotFrench** [comment]
Were you ever able to reproduce this? I haven't seen it happen.

**Comment by paulkim26** [comment]
Will close for now since no repro is available + no one's encountered since. Feel free to re-open if unsolved.

**Comment by wollo77** [comment]
Happened again today, for the first time after a long while:
![image](https://github.com/paulkim26/collab-museum/assets/135688731/fc3a3ed0-845c-4965-b0d1-89ed85e78db9)
